### ISSUE TITLE
Add execution options to schedule and execute commands

### ISSUE THREAD

**Issue by ggalves** [globocom/azkaban-cli#19]
Azkaban schedule command let you pass some execution options. One of them (`concurrent_option`) is already supported.

```python
@login_required
def __schedule(ctx, project, flow, cron, concurrent_option):
    azkaban = ctx.obj[u'azkaban']

    try:
        azkaban.schedule(project, flow, cron, concurrentOption=concurrent_option)
    except ScheduleError as e:
        logging.error(str(e))
```

But others (like `failureAction`) is not supported yet.

All execution options allowed can be found here: https://azkaban.readthedocs.io/en/latest/ajaxApi.html#execute-a-flow

**Comment by hermancaldara** [comment]
I'm also doing it!

**Comment by LeonardoFurtado** [comment]
Hello everyone :D I'm doing it, but we need all execution options? As you said before, concurrent_option and failure_action are interesting to have, but there is 15 execution options on documentation. let me know if we need all before I made a PR. I'll work on another issue in the meantime. 🔨

**Comment by LeonardoFurtado** [comment]
Oh, there is an old PR merged about it #53 . Need to close this issue? @aguimaraesduarte @dmvieira

**Comment by aguimaraesduarte** [comment]
@LeonardoFurtado Yes, sorry. I'm closing this issue now. Thanks for reporting!
